vim.md

```diff
@@ -0,0 +1,196 @@
+# Vim and Markdown
+
+## Vim
+
+Vim is described as a 'highly configurable text editor for efficiently creating
+and changing any kind of text'
+
+Vim is included under the alias vi on most UNIX systems, as well as OS-X
+
+Vim is designed with the intent to have functions easily accessable and to never
+require the user to remove their hands from the home row.
+
+### Getting started with Vim
+
+To start vim without opening a specific file simply type `vim` on the terminal.
+To open a file, write `vim filename`, in the case of this file, one would type
+`vim vim.md`, like you likely already have.
+
+to quit vim, simply hit `esc` a couple of times, then type `:q` and hit enter.
+Hitting escape ensures that you are not in insert mode. if this gives you an
+error, typing `:q!` will force exit, but will discard all changes.
+
+to save a file in vim, go to normal mode and type `:w` this will save the file
+directly back to the same filename. this can be combined with the `q` command
+with `:wq`, saving and quitting at the same time.
+
+#### Modes of Vim
+
+Before we jump into movement, we should quickly go over the 2 modes of vim; 
```

Comment:
There's also normal mode (where you are after pressing `esc` a few times) and an `ed`-like mode (line mode, maybe?). Maybe drop 'the'?